examples/Ray trace Double Gauss/Ray trace double gauss.py

```diff
@@ -0,0 +1,82 @@
+"""
+In standalone runs and plots single ray traces
+and ray fan analysis in Double Gauss lens.
+"""
+
+import zospy as zp
+import matplotlib.pyplot as plt
+import numpy as np
+
+##########################
+## Input
+
+# Number of rays per field
+Nr = 3
+
+# Field coordinates
+fields = [0,10/14,1]
+
+# Plot colors for fields and wavelengths
+cols = ['b','g','r']
+
+##########################
+## Connect to OpticStudio
+print('Connecting to Zemax API ...')
+zos = zp.ZOS()
+zos.wakeup()
+zos.create_new_application()
+oss = zos.get_primary_system()
+
+# Load example file
+file = "Double Gauss 28 degree field.zos"
```

Comment:
```suggestion
file = "Double Gauss 28 degree field.zmx"
```

Reply:
@andibarg In my example folder, I have no *.zos* files, but only *.zmx* (or *.ZDA*). Is this also true for your folder, or did they update the format in newer OpticStudio versions?

Reply:
@LucVV I am using OpticStudio 22.1.2 and for each example I have both _.zos_ and _.zmx_ files in the folder. So, things still work in my version if we change the filename extension to _.zmx_ and we make it backwork compatible, too.